packages/integrations/svelte/package.json

```diff
@@ -53,7 +53,8 @@
   },
   "peerDependencies": {
     "astro": "^4.0.0",
-    "svelte": "^4.0.0 || ^5.0.0-next.56"
+    "svelte": "^4.0.0 || ^5.0.0-next.56",
+    "typescript": "^5.3.3"
```

Comment:
This is not as bad as it might seem when you consider that two of our presets install TypeScript for you anyway.

Reply:
Should we mark it as an optional peer dep? Otherwise it seems like a breaking change

Reply:
I feel like it was more of an oversight to not have it, since not having it result in bugs, so it's like a bug fix in some ways? I'm happy to move it to an optionalDep, though

Reply:
The line is definitely blur to me especially that this dep is only needed for editor tooling, so technically dev/build is not affected. However, package manager installs we get a warning of a missing dep now.

If we're ok with the warnings that users might get, then I think it's not crucial to have it optional. I also don't mind not making this optional if you think this is better overall!

Reply:
Maybe we can do a minor as a middleground? 😄

Reply:
Works for me 👍